src/libcrun/handlers/krun.c

```diff
@@ -293,6 +295,49 @@ libkrun_unload (void *cookie, libcrun_error_t *err arg_unused)
   return 0;
 }
 
+static int
+libkrun_modify_oci_configuration (void *cookie, libcrun_context_t *context,
```

Comment:
Probably `libcrun_oci_inject_device`, since I hope function should not do any more modifications to OCI spec, or maybe `libcrun_oci_inject_kvm` but it is also fine as it is.

Reply:
it is better to keep it generic, so we don't need another hook if the need arises